controllers/volumereplicationgroup_controller.go

```diff
@@ -51,6 +51,17 @@ type VolumeReplicationGroupReconciler struct {
 	Scheme *runtime.Scheme
 }
 
+type VRGInstance struct {
```

Comment:
Is this intentionally a public type or is it a side-effect of using the VRG acronym?

Reply:
These functions may go into a separate file, so that controller file just contains methods of `VolumeReplicationGroupReconciler` and a new file in the same dir contains methods for `VRGInstance`.

It is not needed strictly per-se now.